### PR TITLE
[UPD] Brainbean Apps => CorporateHub

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
cc @pedrobaeza please merge this one

File: web_widget_dropdown_dynamic/README.rst [hunk 1]
```diff
@@ -89,12 +89,14 @@ Credits
...
 ~~~~~~~
 
-* Brainbean Apps OU
+* CorporateHub
 
 Contributors
 ~~~~~~~~~~~~
 
-* Alexey Pelykh <alexey.pelykh@brainbeanapps.com>
+* `CorporateHub <https://corporatehub.eu/>`__
```
Comment (simahawk) [inline web_widget_dropdown_dynamic/README.rst:97]: you should change it in readme/CONTRIBUTORS.rst ;)

Comment (alexey-pelykh) [reply web_widget_dropdown_dynamic/README.rst:97]: @simahawk yeah, I know, but in any case README will be regenerated

Comment (simahawk) [reply web_widget_dropdown_dynamic/README.rst:97]: eh, you changed something that will be overridden w/ the old value :wink:

Comment (simahawk) [reply web_widget_dropdown_dynamic/README.rst:97]: https://github.com/OCA/web/blob/2069abd6b78f5b56212d8ad888a3ee896f3c66e1/web_widget_dropdown_dynamic/readme/CONTRIBUTORS.rst

Comment (alexey-pelykh) [reply web_widget_dropdown_dynamic/README.rst:97]: Oh, that's a good catch! I wonder how I did miss that, thanks

Comment (alexey-pelykh) [reply web_widget_dropdown_dynamic/README.rst:97]: @simahawk fixed, thanks!

**Comment by alexey-pelykh** [comment]
@OCA/web-maintainers would it be possible to merge this?

**Review by yvaucher — APPROVED** [review]

**Pull request merged**